dontAutomergeMajor.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "Disables automerge for major updates.",
+  "major": {
+    "automerge": false,
```

Comment:
This is already the default behaviour in most cases, but there's no harm in being explicit, and it means we can have the same `labels` functionality as other cases